test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1956,6 +1959,28 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
             expect(test.form.el('factor-question').length).toEqual(1);
           });
       });
+      itp('calls transaction.authenticate with username, options and stateToken', function () {
+        return setupPasswordlessAuth(null, true, false).then(function (test) {
+          $.ajax.calls.reset();
+          test.form.setUsername('testuser@test.com');
+          test.form.submit();
+          return tick(test);
```

Comment:
`tick` is bad practice. try to use `waitFor*` utilies (https://github.com/okta/okta-signin-widget/blob/master/test/unit/helpers/util/Expect.js)